hook!-full-game/src/Player/PassThroughDetector.gd

```diff
@@ -0,0 +1,16 @@
+extends Area2D
+
+onready var kinematic_body: KinematicBody2D = get_node(kinematic_body_path) as KinematicBody2D
+export var kinematic_body_path: NodePath = ".."
```

Comment:
`export var kinematic_body_path: = NodePath("..")` to create a nodepath object + type inference, following the guidelines.